setup.cfg

```diff
@@ -31,17 +34,17 @@ project_urls =
 url = https://github.com/reactive-firewall/multicast.git
 
 [bdist_wheel]
-universal=1
+universal=0
```

Comment:
Fix for GH issue #120 - possible only because we are migrating to newer implementations 🎉